src/main/java/use_case/registration/RegistrationInputData.java

```diff
@@ -0,0 +1,4 @@
+package use_case.registration;
+
+public record RegistrationInputData(String username, String password) {
```

Comment:
what is the `record` here? Is the class not yet implemented?
You should have `public class RegistrationInputData{}`
This one should take in the username, password, repeatPassword(if any, check the UI design).

Reply:
@TheGreatCorrine The `record` here is correct. `record` is a easier way to create data class in Java. We are also using it for buy stock use case.